bundles/org.openhab.binding.fmiweather/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,382 @@
+	<channel-type id="time-channel">
+		<item-type>DateTime</item-type>
+		<label>Observation Time</label>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS" />
+	</channel-type>
+	<channel-type id="forecast-time-channel">
+		<item-type>DateTime</item-type>
+		<label>Forecast Time</label>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS" />
+	</channel-type>
+	<channel-type id="temperature-channel">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="humidity-channel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="wind-direction-channel">
+		<item-type>Number:Angle</item-type>
+		<label>Wind Direction</label>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="wind-speed-channel">
+		<item-type>Number:Speed</item-type>
+		<label>Wind Speed</label>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="wind-gust-channel">
+		<item-type>Number:Speed</item-type>
+		<label>Wind Gust</label>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="pressure-channel">
+		<item-type>Number:Pressure</item-type>
+		<label>Pressure</label>
+		<category>Pressure</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="precipitation-channel">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation</label>
+		<description>Precipitation in one hour</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="snow-depth-channel">
+		<item-type>Number:Length</item-type>
+		<label>Snow depth</label>
+		<category>Snow</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="visibility-channel">
+		<item-type>Number:Length</item-type>
+		<label>Visibility</label>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="clouds-channel">
+		<item-type>Number</item-type>
+		<label>Cloudiness</label>
+		<description>Given in numbers out of eight: 0 = clear skies, 8 = overcast, 9 = cloud coverage could not be determined. Takes into account all cloud layers.</description>
+		<state readOnly="true" pattern="%.0f %unit%" min="0" max="9" step="1"/>
+	</channel-type>
+	<channel-type id="present-weather-channel">
+		<item-type>Number</item-type>
+		<label>Prevailing weather</label>
+		<state readOnly="true" min="0" max="99" pattern="%s">
```

Comment:
I guess a short description (like the comment in the xml) will help here a lot :)

Reply:
Fixed in 4d2aadb